app/src/test/java/com/codesoom/assignment/controllers/TaskControllerTest.java

```diff
@@ -0,0 +1,121 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.TaskService;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.*;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@DisplayName("TaskController 테스트")
+class TaskControllerTest {
+    private TaskService taskService;
+    private TaskController taskController;
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+        taskController = new TaskController(taskService);
+
+        Task createTask = new Task();
+        createTask.setTitle("Test1");
+        taskController.create(createTask);
+    }
+
+    @Nested
+    @DisplayName("list 메서드는")
+    class Describe_list {
+        @Test
+        @DisplayName("Task 리스트를 반환한다")
+        void itReturnsListOfTask() {
+            Assertions.assertEquals(taskController.list().size(), 1, "리턴된 Task 리스트는 사이즈 값으로 1을 갖는다");
```

Comment:
마지막의 메시지는 테스트 조건에 실패했을 때 출력되는 메시지가 들어가야 합니다. 

![image](https://user-images.githubusercontent.com/1855714/107039849-c425ce00-6801-11eb-83bb-fa3dbc587d41.png)

```suggestion
            Assertions.assertEquals(taskController.list().size(), 1, "리턴된 Task 리스트의 사이즈가 1이어야 합니다.");
```
이렇게 테스트가 실패했을 경우, 실패한 원인을 알려주도록 작성해주세요.

Reply:
`Fails with the supplied failure {@code message}` 라는 의미가 테스트 실패의 경우 실패 원인을 쓰는 것이었군요!

Reply:
네 항상 다른 사람이 만든 코드를 사용할 때, 주석을 확인하는 습관을 기르세요.

Reply:
넵 :)